share/OpenMS/SCHEMAS/IdXML_1_4.xsd

```diff
@@ -249,7 +249,7 @@
 				<xs:documentation>significance threshold as calculated by the search engine</xs:documentation>
 			</xs:annotation>
 		</xs:attribute>
-		<xs:attribute name="spectrum_reference" type="xs:unsignedInt">
+                <xs:attribute name="spectrum_reference" type="xs:string">

```

Comment:
seems like it is not backward compatible, e.g. if there is an 1.4 file that has a string it cannot be read by a parser that uses the old 1.4 so there should be a 1.5 version imho